init/devices.cpp

```diff
@@ -62,13 +62,17 @@ namespace init {
 const int Aliases::ANY = -1;
 
 std::string Aliases::ToString() const {
-    std::stringstream fmt;
+    std::stringstream fmt, major_hex, minor_hex;
+
+    major_hex << std::hex << major_;
+    minor_hex << std::hex << minor_;
 
-    fmt << "AliasTo:" << alias_to_ << " "
-        << "Major:" << std::hex << (major_ == Aliases::ANY ? "Any" : std::to_string(major_)) << " "
-        << "Minor:" << std::hex << (minor_ == Aliases::ANY ? "Any" : std::to_string(minor_)) << " "
-        << "ProductId:" << std::hex << productId_ << " "
-        << "VendorId:" << std::hex << vendorId_;
+    fmt << std::hex
+        << "AliasTo:" << alias_to_ << " "
+        << "Major:" << (major_ == Aliases::ANY ? "Any" : major_hex.str()) << " "
+        << "Minor:" << (minor_ == Aliases::ANY ? "Any" : minor_hex.str()) << " "
```

Comment:
I don't think major, minor have to be in hex. Unlike the USB IDs these aren't generally reported in hex by `ls` or in `/proc/devices` AFAIK. Anyways, no big deal.